src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -58,6 +58,7 @@ class PhpDumper extends Dumper
     private $targetDirRegex;
     private $targetDirMaxMatches;
     private $docStar;
+    private $existingNames = array();
```

Comment:
Sorry for bothering you again about this ... but after looking at the code with more calm, I think the "right" name for this variable would be `$serviceNames` instead of `$existingNames`. Thanks!